### PR TITLE
k3s-2022-Annual Review

### PR DESCRIPTION
**Pull request by cwayne18**
2022 annual review for K3s

### CODE REVIEW

**Comment by jasonbraganza** [comment]
Hello, I’ll take a run at reviewing the report.

**Comment by jasonbraganza** [comment]
The TAG recommends the TOC continue sponsorship.

The TAG further recommends K3S bring about the following changes: 
1. Update [ROADMAP.md ](https://github.com/k3s-io/k3s/blob/master/ROADMAP.md)and bring it up to the project’s current path
2. Include a note on the project’s plans on contributing changes back upstream to Kubernetes ([that addresses the second point of encouragement here](https://lists.cncf.io/g/cncf-toc/message/5081))
    - Have any changes been contributed upstream?
    - Are there plans to do so, going forward?
    - If there aren’t any, then to explain / justify the same.
3. And while not a requirement for sandbox projects, K3s implements a [GOVERNANCE.md](https://github.com/cncf/project-template/blob/main/GOVERNANCE-maintainer.md) that captures the basics of decision making within the project, especially to progress towards Incubation at some point.

File: reviews/2022-k3s-annual.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+| [GitHub Stars](https://github.com/k3s-io/k3s/stargazers) | ~14k | ~18k | ~21k | 
+| [Commits](https://github.com/k3s-io/k3s/commits/master) | ~75/month | ~65/month | [~100/month](https://github.com/k3s-io/k3s/pulse/monthly)  |
+| [Merged PRs](https://github.com/k3s-io/k3s/pulls) | ~35/month | ~45/month | [~66/month](https://github.com/k3s-io/k3s/pulse/monthly)  | 
+| [Releases](https://github.com/k3s-io/k3s/releases) | 100 | 171 | 189 |
+
+### How many maintainers do you have, and which organizations are they from?
+
+Our maintainers list: https://github.com/k3s-io/k3s/blob/master/MAINTAINERS
+
+Currently, the project has 17 maintainers, all with SUSE:
```
Comment (cathyhongzhang) [inline reviews/2022-k3s-annual.md:92]: To build a healthy and sustainable community, you need to have maintainers from more than one company.

**Comment by amye** [comment]
Marking this as complete, annual reviews are no longer a requirement for sandbox. 
We'll be rolling out more efficient ways to show project health and metrics in 2024.

**Pull request merged**